### PR TITLE
fix: exclude the test file from bundle

### PR DESCRIPTION
**Pull request by Gehbt**
on v1.12.0
image

on v1.12.1   with .test file
image

### CODE REVIEW

**Review by lukastaegert — APPROVED** [review]
While I am not sure why it was not included before, this definitely fixes this.

**Comment by lukastaegert** [comment]
Could you please run `npm run lint`?

File: tests/no-side-effects-in-initialization.test.ts [hunk 1]
```diff
@@ -1,10 +1,10 @@
-/* Possible improvements for rollup:
- * * Properly handle valid this values
- * * Deleting an object expression member whose value is a global does not need to be a side-effect
+/** Possible improvements for rollup:
+ * Properly handle valid this values
+ * Deleting an object expression member whose value is a global does not need to be a side-effect
```
Comment (lukastaegert) [inline tests/no-side-effects-in-initialization.test.ts:3]: This is actually supposed to be a bulleted list. If you want to improve its looks, maybe change it to

```suggestion
/* Possible improvements for rollup:
 * - Properly handle valid this values
 * - Deleting an object expression member whose value is a global does not need to be a side-effect
```

Comment (Gehbt) [reply tests/no-side-effects-in-initialization.test.ts:3]: Yes, it's my misunderstanding.

Comment (Gehbt) [reply tests/no-side-effects-in-initialization.test.ts:3]: Yes, it's my misunderstanding.

**Pull request merged**